src/BuildingRegistry.Api.CrabImport/Infrastructure/Modules/MediatRModule.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace BuildingRegistry.Api.CrabImport.Infrastructure.Modules
+{
+    using System.Reflection;
+    using Autofac;
+    using Handlers;
+    using MediatR;
+    using Module = Autofac.Module;
+
+    public class MediatRModule : Module
+    {
+        protected override void Load(ContainerBuilder builder)
+        {
+            builder
+                .RegisterType<Mediator>()
+                .As<IMediator>()
+                .InstancePerLifetimeScope();
+
+            // request & notification handlers
```

Comment:
this block should not be there anymore (MediatR 12.0.1)

Reply:
it's still 10.x